shipstation_integration/shipments.py

```diff
@@ -158,3 +156,35 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+def create_shipment(shipment, delivery_note, store):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.description_of_content = "Farm plastic"
```

Comment:
Add a todo here to change this to include all shipment items instead. This is fine for the first draft.